### PR TITLE
[C-830] Move account to common store and set up with sign in

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Moves signon slice to common and includes the store in mobile. Sends some actions there, but with the sagas disabled, does nothing effectively.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Web + mobile

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/common/services/audius-backend/AudiusBackend.ts [hunk 1]
```diff
@@ -590,6 +590,7 @@ export const audiusBackend = ({
     )
     const { ethWeb3Config } = getEthWeb3Config()
     const { solanaWeb3Config } = getSolanaWeb3Config()
+    console.log({ solanaWeb3Config })
```
Comment (dylanjeffers) [inline packages/web/src/common/services/audius-backend/AudiusBackend.ts:593]: log

File: packages/web/src/pages/sign-on/SignOn.tsx [hunk 2]
```diff
@@ -6,16 +6,17 @@ import { withRouter, RouteComponentProps } from 'react-router-dom'
 import { Dispatch } from 'redux'
 
 import { getHasAccount } from 'common/store/account/selectors'
-import { Pages } from 'pages/sign-on/store/types'
+import { Pages } from 'common/store/pages/signon/types'
 import { AppState } from 'store/types'
 import { isMobile } from 'utils/clientUtil'
 import { TRENDING_PAGE } from 'utils/route'
 
+import * as signOnAction from '../../common/store/pages/signon/actions'
```
Comment (dylanjeffers) [inline packages/web/src/pages/sign-on/SignOn.tsx:14]: absolute imports

File: packages/web/src/pages/sign-on/SignOnProvider.tsx [hunk 3]
```diff
@@ -32,6 +32,15 @@ import {
...
 } from 'utils/route'
 
+import * as signOnAction from '../../common/store/pages/signon/actions'
+import {
+  getSignOn,
+  getIsMobileSignOnVisible,
+  getToastText,
+  makeGetFollowArtists,
+  getRouteOnExit
+} from '../../common/store/pages/signon/selectors'
```
Comment (dylanjeffers) [inline packages/web/src/pages/sign-on/SignOnProvider.tsx:42]: imports

File: packages/web/src/pages/sign-on/components/desktop/FollowPage.tsx [hunk 4]
```diff
@@ -8,7 +8,10 @@ import { ReactComponent as IconWand } from 'assets/img/iconWand.svg'
 import UserCard from 'components/card/UserCard'
 import SelectablePills from 'components/selectable-pill/SelectablePills'
 
-import { FollowArtistsCategory, artistCategories } from '../../store/types'
+import {
+  FollowArtistsCategory,
+  artistCategories
+} from '../../../../common/store/pages/signon/types'
```
Comment (dylanjeffers) [inline packages/web/src/pages/sign-on/components/desktop/FollowPage.tsx:14]: imports

File: packages/web/src/pages/sign-on/components/mobile/FollowPage.tsx [hunk 5]
```diff
@@ -9,7 +9,10 @@ import UserCard from 'components/card/UserCard'
 import SelectablePills from 'components/selectable-pill/SelectablePills'
 import { MAIN_CONTENT_ID } from 'pages/App'
 
-import { FollowArtistsCategory, artistCategories } from '../../store/types'
+import {
+  FollowArtistsCategory,
+  artistCategories
+} from '../../../../common/store/pages/signon/types'
```
Comment (dylanjeffers) [inline packages/web/src/pages/sign-on/components/mobile/FollowPage.tsx:15]: sameee

File: packages/mobile/src/store/store.ts [hunk 6]
```diff
@@ -41,7 +41,7 @@ export type AppState = {
   oauth: OAuthState
   remoteConfig: RemoteConfigState
   search: SearchState
-  signon: SignonState
+  signonDeprecated: SignonState
```
Comment (dylanjeffers) [inline packages/mobile/src/store/store.ts:44]: nice idea, lets just use this pattern going forward?

File: packages/mobile/src/store/sagas.ts [hunk 7]
```diff
@@ -1,4 +1,5 @@
-// import accountSagas from 'common/store/account/sagas'
+import signOnSagas from 'audius-client/src/common/store/pages/signon/sagas'
```
Comment (dylanjeffers) [inline packages/mobile/src/store/sagas.ts:1]: a bit weird, but i think we should use common/store, and not includi audius-client, we have the alias set up to work that way

File: packages/web/src/pages/sign-on/SignOn.tsx (hunk 2 above)
Comment (raymondjacobson) [inline packages/web/src/pages/sign-on/SignOn.tsx:14]: fixing

File: packages/mobile/src/store/sagas.ts (hunk 7 above)
Comment (dylanjeffers) [inline packages/mobile/src/store/sagas.ts:1]: also is the goal to keep these imported and not user them below? suppose that is an easier flow and doesn't hurt anything?

File: packages/mobile/src/screens/signon/SignOn.tsx [hunk 8]
```diff
@@ -665,6 +666,9 @@ const SignOn = ({ navigation }: SignOnProps) => {
   }
 
   const validateEmail = (email: string) => {
+    dispatch(signOnActionsWeb.setValueField('email', email))
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/signon/SignOn.tsx:669]: as far as pattern, do we want this to just be `signOnActions` and the other one be `signOnLegacyActions`, to follow pattern in the store?

**Review by dylanjeffers — APPROVED** [review]
real awesome job

File: packages/mobile/src/screens/signon/SignOn.tsx (hunk 8 above)
Comment (raymondjacobson) [reply packages/mobile/src/screens/signon/SignOn.tsx:669]: I called it signOnDeprecated, but that sounds better, ill switch everything to signOnLegacy

File: packages/mobile/src/store/sagas.ts (hunk 7 above)
Comment (raymondjacobson) [reply packages/mobile/src/store/sagas.ts]: removed

File: packages/mobile/src/store/store.ts (hunk 6 above)
Comment (raymondjacobson) [reply packages/mobile/src/store/store.ts]: renamed legacy

File: packages/mobile/src/store/sagas.ts (hunk 7 above)
Comment (raymondjacobson) [reply packages/mobile/src/store/sagas.ts]: defer until rebase

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-account-sagas

**Pull request merged**